src/main/Config.cpp

```diff
@@ -1082,16 +1081,20 @@ Config::processConfig(std::shared_ptr<cpptoml::table> t)
                          "is ignored. Please remove from config");
                  }},
                 {"DEPRECATED_SQL_LEDGER_STATE",
-                 [&]() { DEPRECATED_SQL_LEDGER_STATE = readBool(item); }},
+                 [&]() {
+                     CLOG_WARNING(
+                         Bucket,
+                         "DEPRECATED_SQL_LEDGER_STATE is deprecated and "
+                         "ignored. Please remove from config");
+                 }},
                 // Still support EXPERIMENTAL_BUCKETLIST_DB* flags for
                 // captive-core for 21.0 release, remove in 21.1 release
                 {"EXPERIMENTAL_BUCKETLIST_DB",
                  [&]() {
-                     DEPRECATED_SQL_LEDGER_STATE = !readBool(item);
                      CLOG_WARNING(
                          Bucket,
-                         "EXPERIMENTAL_BUCKETLIST_DB flag is deprecated, "
-                         "use DEPRECATED_SQL_LEDGER_STATE=false instead.");
+                         "EXPERIMENTAL_BUCKETLIST_DB flag is deprecated. "
```

Comment:
the comment says  EXPERIMENTAL_BUCKETLIST_DB can be removed after v21.1, can we remove those or is the comment inaccurate?

Reply:
Given that Horizon only wants to remove flags in protocol upgrades, this will have to stick around until v23. I'll update the comment accordingly and link an issue.

Reply:
thanks. let's stick with issues then rather than a TODO (as it suggests unfinished code)